### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/ba080c313c09c38e4fc26ba427fd13e9c66385d3

### CODE REVIEW

File: en/firefox/all.ftl [hunk 1]
```diff
@@ -4,31 +4,65 @@
...
+firefox-all-down-arrow = Choose from the list below
 
 # Used as an accessible label for a help button. The text is replaced with a "?" icon.
 firefox-all-get-help = Get help
-
-firefox-all-you-are-about-to-download = You are about to download:
-firefox-all-browser = Browser:
-firefox-all-platform = Platform:
-firefox-all-language = Language:
+firefox-all-browser = 1. Browser:
```
Comment (peiying2) [inline en/firefox/all.ftl:32]: Make string ID `v2`

File: en/firefox/all.ftl [hunk 2]
```diff
@@ -4,31 +4,65 @@
...
+firefox-all-platform = 2. Platform:
```
Comment (peiying2) [inline en/firefox/all.ftl:33]: Make ID `v2`

File: en/firefox/all.ftl [hunk 3]
```diff
@@ -4,31 +4,65 @@
...
+firefox-all-language = 3. Language:
```
Comment (peiying2) [inline en/firefox/all.ftl:34]: Make string ID `v2`

**Review by peiying2 — CHANGES_REQUESTED** [review]
Need to update string IDs for revised strings.

**Comment by stephaniehobson** [comment]
https://github.com/mozilla/bedrock/pull/15033

**Comment by peiying2** [comment]
> [mozilla/bedrock#15033](https://github.com/mozilla/bedrock/pull/15033)

@stephaniehobson Thank you for the fix. Do you want to label the `obsolete` strings and add a calendar date to them?

**Review by peiying2 — APPROVED** [review]
@stephaniehobson I approve the PR but please add the comments to the obsolete strings when you get a chance.

**Comment by stephaniehobson** [comment]
Obsolete string dated: https://github.com/mozilla/bedrock/pull/15033

**Pull request merged**